Framework/src/SliceTrendingTask.cxx

```diff
@@ -543,3 +546,87 @@ void SliceTrendingTask::beautifyGraph(T& graph, const SliceTrendingTaskConfig::P
     graph->GetXaxis()->SetNoExponent(true);
   }
 }
+
+void SliceTrendingTask::beautifyLegend(TLegend* leg, const SliceTrendingTaskConfig::Plot& plotconfig, TCanvas* canv)
+{
+  int ncolums = 2;
+  try {
+    ncolums = std::stoi(plotconfig.legendNColums);
+  } catch (...) {
+    LOG(error) << "key legNColums must be integer" << ENDM;
```

Comment:
in the framework we use the infologger, could you switch to `ILOG` ?

Reply:
Done